app/service/utility_svc.py

```diff
@@ -13,7 +13,7 @@ def apply_stealth(executor, code):
 
     @staticmethod
     def decode_bytes(s):
-        return b64decode(s).decode('utf-8')
+        return b64decode(s).decode('utf-8').replace("\n","")
```

Comment:
good change. add single quotes.

Reply:
after discussing with @ArtificialErmine we think this should take care of VIRTS-251 too.

Reply:
That one gets me every time.

Reply:
great!!